pkg/executor/aggfuncs/spill_serialize_helper.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs
+
+import (
+	"unsafe"
+
+	"github.com/pingcap/tidb/pkg/types"
+	"github.com/pingcap/tidb/pkg/util/spill"
+)
+
+// spillSerializeHelper can only be used for one aggregator function.
+// It may cause error if agg func1 and agg func2 use the same spillSerializeHelper.
+type spillSerializeHelper struct {
+	buf []byte
+}
```

Comment:
If we only need the `buf` attribute, I think the definition of `spillSerializeHelper` is not necessary.
`buf` can be defined as the input argument of `serializePartialResult4Count`, etc.

Reply:
> If we only need the `buf` attribute, I think the definition of `spillSerializeHelper` is not necessary. `buf` can be defined as the input argument of `serializePartialResult4Count`, etc.

Serialization of spill for agg may have to handle more complexity situation in the future, so I think it's better to define a `helper` to tackle possible intractable problems.